apps/judicial-system/backend/src/app/modules/notification/guards/rolesRules.ts

```diff
@@ -28,20 +28,28 @@ export const defenderNotificationRule: RolesRule = {
   ],
 } as RolesRule
 
-// Allows district court judges to send notifiications
+// Allows district court judges to send notifications
 export const districtCourtJudgeNotificationRule: RolesRule = {
   role: UserRole.DISTRICT_COURT_JUDGE,
   type: RulesType.FIELD_VALUES,
   dtoField: 'type',
-  dtoFieldValues: [CaseNotificationType.COURT_DATE],
+  dtoFieldValues: [
+    CaseNotificationType.COURT_DATE,
+    CaseNotificationType.DISTRICT_COURT_JUDGE_ASSIGNED,
+    CaseNotificationType.DISTRICT_COURT_REGISTRAR_ASSIGNED,
+  ],
```

Comment:
Please remove the new notification types from roles rules. These roles rules are only used in client facing controller endpoints. Since these notifications do not come from the client, but rather the case service, users should not be given permission to send them.